Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -145,7 +145,7 @@
 
 - type: cargoBounty
   id: BountyFigurine
-  reward: 8000
+  reward: 1600
```

Comment:
Figurine crate costs 4000 for enough to fufill this bounty twice. Can't make figurines on station, would also mean either skipping or taking a loss every time it shows up.

Reply:
> Figurine crate costs 4000 for enough to fufill this bounty twice. Can't make figurines on station, would also mean either skipping or taking a loss every time it shows up.

what do you mean "cant make figurines on station", you can find it at:

- salvage wrecks
- art toolboxes
- Good Clean Fun vending

Reply:
Thats not making them, thats taking them away from antags(thieves) and crew.

Reply:
so art toolboxes have 1 and the good clean fun vending has 2 (3?) this means that unless you manage to find other figurines on the station (which isn't impossible, but is map dependent) this bounty can, at best, be completed once (maybe twice, map dependent) for a profit without relying on salv to get lucky. Why on gods green earth would you make it this way. Just because its "possible" to complete while wringing the station dry doesn't mean it needs to be that way? If you want to make cargo completely reliant on salv just remove the bounty system entirely, buff the sell prices of maints loot for salv to sell, and relegate cargo techs to desk work.

The entire point of the bounty system is so that cargo can **SOMEWHAT** function without salv. Sure some of the bounties need salvs help, but it shouldn't be all of them consistently.